### PR TITLE
Create unit test sort book

### PR DESCRIPTION
**Pull request by at-luanle**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: #PBI-7
- Task ID: #PBI-7.3

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
tests/Browser/SortBookTest.php
-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-nghiavo** [comment]
lgtm

File: tests/Browser/Pages/SortBookTest.php [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+
+    /**
+     * A User with role 1 test example.
+     *
+     * @return void
+     */
+    public function userLogin()
+    {
+        factory(User::class, 1)->create([
+            'employee_code' => 'ATI0285',
```
Comment (at-huydq) [inline tests/Browser/Pages/SortBookTest.php]: ko dùng employee_code, tên vs email của cty. Mấy cái này dùng faker tạo hết đi.

**Review by ghost — COMMENTED** [review]
please check my comment

**Comment by at-huydq** [comment]
LGTM

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**